main.tf

```diff
@@ -8,13 +8,7 @@ resource "azurerm_resource_group" "default" {
   name     = var.namespace
   location = var.location
 
-  tags = merge(
-    {
-      "customer-ns" = var.namespace,
-      "env"         = "managed-install"
-    },
-    var.tags,
-  )
+  tags = var.tags
```

Comment:
> removed the "customer-ns" and "env" tags

could you please share if these tags will be set somewhere else, or are we just removing them altogether? If the intention is to remove it altogether, are we just removing these tags from cloud infra, or also from observability?

Reply:
We need to pass the tags to the module on invocation, instead of hard-coding them into the repo. This is just cleanup; fixing tags in this repo is in our backlog.

Reply:
Essentially, because this is a public facing repo, we don't want w&b specific tags to be set.  That should be handled in our code that references this module instead.